connect/values.yaml

```diff
@@ -28,7 +28,7 @@ operator:
   pollingInterval: 10
   version: "1.0.0"
   nodeSelector: {}
-  watchNamespace: []
+  watchNamespace: ["{{ .Release.Namespace }}"] # This will be interpolated in the templates, using the tpl function
```

Comment:
Since this is no longer an empty array we should move it back to a yaml array

```
watchNamespace:
  - {{ .Release.Namespace }}  # This will be interpolated in the templates, using the tpl function
```